chromium_src/chrome/browser/printing/print_view_manager_base.h

```diff
@@ -159,6 +166,8 @@ class PrintViewManagerBase : public content::NotificationObserver,
 
   scoped_refptr<printing::PrintQueriesQueue> queue_;
 
+  std::unique_ptr<base::Callback<void(bool)>> callback_ptr;
```

Comment:
Callbacks are usually saved by value, and passed by const reference, it is preferred to declare it as `base::Callback<void(bool)> callback_`. You can check whether it is null with `callback_.is_null()`.

It is mainly because passing a callback is very cheap, especially compared to a heap allocation.